pkg/ddc/jindofsx/transform.go

```diff
@@ -211,22 +211,33 @@ func (e *JindoFSxEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, met
 	propertiesFileStore := map[string]string{}
 
 	for _, mount := range dataset.Spec.Mounts {
-		if !strings.HasSuffix(mount.MountPoint, "/") {
-			mount.MountPoint = mount.MountPoint + "/"
-		}
 		// support nas storage
-		if strings.HasPrefix(mount.MountPoint, "local:///") {
-			value.Mounts.Master[mount.Name] = &Level{
-				Path: mount.MountPoint[8:],
-				Type: "hostPath",
+		if strings.HasPrefix(mount.MountPoint, "pvc://") {
+			if len(value.UFSVolumes) == 0 {
+				value.UFSVolumes = []UFSVolume{}
+			}
+			ufsVolumesName := strings.TrimPrefix(mount.MountPoint, common.VolumeScheme.String())
+			ufsVolumesPath := utils.UFSPathBuilder{}.GenLocalStoragePath(mount)
+			value.UFSVolumes = append(value.UFSVolumes, UFSVolume{
+				Name:          ufsVolumesName,
+				ContainerPath: ufsVolumesPath,
+			})
+		} else {
+			if !strings.HasSuffix(mount.MountPoint, "/") {
+				mount.MountPoint = mount.MountPoint + "/"
 			}
-			value.Mounts.WorkersAndClients[mount.Name] = &Level{
-				Path: mount.MountPoint[8:],
-				Type: "hostPath",
+			if strings.HasPrefix(mount.MountPoint, "local:///") {
+				value.Mounts.Master[mount.Name] = &Level{
+					Path: mount.MountPoint[8:],
+					Type: "hostPath",
+				}
+				value.Mounts.WorkersAndClients[mount.Name] = &Level{
```

Comment:
Agree with @cheyang .I think no need to mount hostPath to Jindofsx's fuse pod.

Reply:
fuse also need to access to pvc as client

Reply:
Is it necessary? I think if the master and worker can access the pvc, it's enough. I think the fuse client can delegate the access to the master and workers.

Reply:
NO,jindo is not like alluxio ,it needs client to access to pvc, alluxio read through worker, that's different